test/conformance/testdata/ineligible_endpoints.yaml

```diff
@@ -415,21 +238,9 @@
 - endpoint: logFileListHandler
   reason: optional feature
   link: https://github.com/kubernetes/kubernetes/issues/108675
-- endpoint: readNetworkingV1NamespacedNetworkPolicyStatus
-  reason: endpoints is currently feature gated and and will only receive e2e & conformance test in 1.25
-  link: https://github.com/kubernetes/kubernetes/pull/107963
 - endpoint: patchNetworkingV1NamespacedNetworkPolicyStatus
   reason: endpoints is currently feature gated and and will only receive e2e & conformance test in 1.25
   link: https://github.com/kubernetes/kubernetes/pull/107963
-- endpoint: replaceNetworkingV1NamespacedNetworkPolicyStatus
-  reason: endpoints is currently feature gated and and will only receive e2e & conformance test in 1.25
-  link: https://github.com/kubernetes/kubernetes/pull/107963
```

Comment:
network policy status is still alpha, so ineligible

Reply:
@liggitt thank you for the feedback. 
My thought was to keep only enpoints in the ineligible_endpoints.yaml that would never be eligble for conformance. Then we move all endpoints that would be tested, including endpoints in alpha or beta that is showing up in APSnoop, in the pending_eligible_endpoints.yaml list. That would mean that ineligible_endpoints.yaml should be mostly settled and endpoints only get moved in and out of pending_eligible_endpoints.yaml as needed.
What could be helpful is to add comments to endpoints in the pending_eligible_endpoints.yaml file to indicate their status. WDYT?

Reply:
ah, I misunderstood the meaning of the new file

> What could be helpful is to add comments to endpoints in the pending_eligible_endpoints.yaml file to indicate their status. WDYT?

that would definitely be helpful... comments on all the `get*APIGroup` endpoints waiting for promotion would be useful

There's a bunch of endpoints in that file that are eligible now (all the `V1` ones, along with `getStorageAPIGroup`). Shouldn't those move straight to eligible instead of pending_eligible?

Reply:
If we take them of the list we will again have a big gap in conformance of untested endpoints. To keep it clean and preserve the history we would pick work from the pending_eligible_endpoints.yaml write test for them all the way to conformance promtion. 
Once that is done we would remove spesific endpoint of the pending_eligible_endpoints.yaml file.
We discussed the method in the last sig arch meeting and there was general greement on the approach.
Adding a description to the endpoints could help to clarify. 
We also update APISnoop to be able to see the graph with or without the pending_eligible_endpoints.yaml and ineligble_endpoints.yaml
https://apisnoop.cncf.io/?exclude-ineligible=true&exclude-pending=true

Reply:
I'm not sure the current 100% indicator communicates accurately... there are presently GA, eligible-for-conformance endpoints which do not have conformance tests, but I don't feel strongly either way

grouping/commenting in the pending_eligible_endpoints.yaml file about which endpoints are eligible now, and which are waiting for an API to graduate would be helpful, but I wouldn't block this PR on that